tools/aapt/Android.mk

```diff
@@ -109,6 +109,46 @@ LOCAL_STATIC_LIBRARIES_windows := $(aaptHostStaticLibs_windows)
 
 include $(BUILD_HOST_EXECUTABLE)
 
+# ==========================================================
+# Build the host static library: libaapt
+# ==========================================================
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := libaapt
+#LOCAL_MODULE_HOST_OS := darwin linux windows
```

Comment:
Could you remove commented out configs? Want to keep patches minimal for easy upgrades to AOSP.